scipy/special/__init__.py

```diff
@@ -797,7 +799,13 @@
 # Deprecated namespaces, to be removed in v2.0.0
 from . import add_newdocs, basic, orthogonal, specfun, sf_error, spfun_stats
 
+
+btdtr = _ufuncs.betainc
+btdtri = _ufuncs.betaincinv
```

Comment:
Should we deprecate these rather than carry two sets of names for the same thing, either here or in a follow-up PR?

Reply:
Done.